SCClassLibrary/Common/Files/SoundFile.sc

```diff
@@ -440,7 +440,7 @@ SoundFile {
 					}, "/n_end", server.addr, nil, ev[\id][0]).oneShot;
 				};
 				if (playNow) {
-					packet = server.makeBundle(false, {ev.play})[0];
+					packet = server.makeBundle(false, {ev.value.play})[0];
```

Comment:
The first argument to `makeBundle` is time: the old `false` was wrong.

While you are at it: minor formatting thing - `{ ev.value.play }` (add spaces).